lib/plugins/Tests/Tests.js

```diff
@@ -0,0 +1,46 @@
+'use strict';
+
+const path = require('path');
+const fs = require('fs');
+
+class Tests {
+  constructor() {
+    this.commands = {
+      test: {
+        commands: {
+          integration: {
+            usage: 'Command for integration testing.',
+            lifeCycleEvents: [
+              'createFile',
+            ],
+          },
+        },
+      },
+    };
+
+    this.hooks = {
+      'test:integration:createFile': this.createFile,
+    };
+  }
+
+  createFile() {
```

Comment:
But if I understand their test correctly they write a package.json that gets then read when they execute npm outdated, so the file is written in preparation, not as a result of the test.

To run and read the output exec seems to work like this: http://stackoverflow.com/questions/12941083/get-the-output-of-a-shell-command-in-node-js